src/elements/footer-block.html

```diff
@@ -51,14 +51,14 @@
 
     </style>
 
-    <div class="container">
+    <footer class="container">
```

Comment:
https://dequeuniversity.com/rules/axe/4.1/region - all page content must be contained by landmarks. `footer` is a landmark/role type.